### PR TITLE
add: stardust-black-hole

### PR DESCRIPTION
**Pull request by Owen-sz**

### CODE REVIEW

File: anda/stardust/black-hole/stardust-black-hole.spec [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+%global shortcommit %(c=%{commit}; echo ${c:0:7})
+# Exclude input files from mangling
+%global __brp_mangle_shebangs_exclude_from ^/usr/src/.*$ 
+
+Name:           stardust-black-hole
+Version:        %commit_date.%shortcommit
+Release:        1%?dist
+Summary:        Spatial storage for stardust xr.
+URL:            https://github.com/StardustXR/black-hole
+Source0:        https://github.com/StardustXR/black-hole/archive/%commit/black-hole-%commit.tar.gz
```
Comment (madonuko) [inline anda/stardust/black-hole/stardust-black-hole.spec]: dry

File: anda/stardust/black-hole/stardust-black-hole.spec [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+License:        MIT
+BuildRequires:  cargo cmake anda-srpm-macros cargo-rpm-macros mold
+Requires:       libgcc glibc
+
+Provides:       black-hole
+
+%description
+Spatial storage for stardust xr
```
Comment (madonuko) [inline anda/stardust/black-hole/stardust-black-hole.spec]: dry and full stop

File: anda/stardust/black-hole/stardust-black-hole.spec [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+Name:           stardust-black-hole
+Version:        %commit_date.%shortcommit
+Release:        1%?dist
+Summary:        Spatial storage for stardust xr.
+URL:            https://github.com/StardustXR/black-hole
+Source0:        https://github.com/StardustXR/black-hole/archive/%commit/black-hole-%commit.tar.gz
+License:        MIT
+BuildRequires:  cargo cmake anda-srpm-macros cargo-rpm-macros mold
+Requires:       libgcc glibc
```
Comment (madonuko) [inline anda/stardust/black-hole/stardust-black-hole.spec]: don't need this

File: anda/stardust/black-hole/stardust-black-hole.spec [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+%description
+Spatial storage for stardust xr
+
+%prep
+%autosetup -n black-hole-%commit
+%cargo_prep_online
+
+%build
+%cargo_build
```
Comment (madonuko) [inline anda/stardust/black-hole/stardust-black-hole.spec]: don't need this

File: anda/stardust/black-hole/update.rhai [hunk 5]
```diff
@@ -0,0 +1,7 @@
+if filters.contains("nightly") {
```
Comment (madonuko) [inline anda/stardust/black-hole/update.rhai]: don't need this

**Review by madonuko — COMMENTED** [review]
add `Packager`

also you assigned me instead of request review

File: anda/stardust/black-hole/stardust-black-hole.spec (hunk 3 above)
Comment (madonuko) [reply anda/stardust/black-hole/stardust-black-hole.spec]: ?

File: anda/stardust/black-hole/update.rhai (hunk 5 above)
Comment (madonuko) [reply anda/stardust/black-hole/update.rhai]: ?

**Review by madonuko — CHANGES_REQUESTED** [review]

File: anda/stardust/black-hole/stardust-black-hole.spec (hunk 3 above)
Comment (Owen-sz) [reply anda/stardust/black-hole/stardust-black-hole.spec]: Removing now

File: anda/stardust/black-hole/stardust-black-hole.spec (hunk 2 above)
Comment (Owen-sz) [reply anda/stardust/black-hole/stardust-black-hole.spec]: What do you mean?

**Review by madonuko — DISMISSED** [review]

**Review by madonuko — DISMISSED** [review]

**Review by madonuko — DISMISSED** [review]

**Review by madonuko — APPROVED** [review]

**Comment by raboneko** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|f39|[](https://github.com/terrapkg/packages/pull/2091)|
|✅|f40|[](https://github.com/terrapkg/packages/pull/2092)|
|✅|f41|[](https://github.com/terrapkg/packages/pull/2093)|
|✅|el9|[](https://github.com/terrapkg/packages/pull/2094)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport) and see the [Github Action logs](https://github.com/terrapkg/packages/actions/runs/10943068916) for details

**Pull request merged**